client-app/package.json

```diff
@@ -34,7 +34,7 @@
   "scripts": {
     "predeploy": "npm run build",
     "deploy": "gh-pages -d build",
-    "start": "react-scripts start --openssl-legacy-provider",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
This may have been committed accidentally, but this is a regression from a previous change

`start` should be before the `--openssl-legacy-provider` flag

Reply:
Resolved during an in-person meeting. We determined it was caused by your node version being out of date.